priv/repo/migrations/20180522223601_create_daily_geo_subdivs.exs

```diff
@@ -0,0 +1,58 @@
+defmodule Castle.Repo.Migrations.CreateDailyGeoSubdivs do
+  use Ecto.Migration
+
+  def change do
+    create table(:daily_geo_subdivs, primary_key: false) do
+      add :podcast_id, :integer, null: false
+      add :episode_id, :uuid, null: false
+      add :country_iso_code, :string, null: false
+      add :subdivision_1_iso_code, :string, null: false
+      add :day, :date, null: false
+      add :count, :integer, null: false
+    end
+
+    execute """
+    CREATE OR REPLACE FUNCTION create_daily_geo_subdivs_partition() RETURNS trigger AS
```

Comment:
Ended up following the pattern of `hourly_downloads` with all these.  Create monthly-partitions, with a trigger to auto upsert into the correct one.